### DISCUSSION TITLE
What is typical RAM usage while idle?

### DISCUSSION THREAD

**Discussion by Travillion** [LizardByte/community#392] [Sunshine] [answered]
I just started using Sunshine with Moonlight on a new system build. I see that Sunshine is using 2 GB of RAM while idle (that is, not streaming, not running anything). The specific service using the RAM seems to be related to the WSL. Is this typical behavior or should I be troubleshooting. That seems like a high draw to have on my system constantly. I hoped to keep Sunshine running always to simplify use (e.g., don't want to have to manually start Sunshine before every session), but I don't think I will if it always takes that much RAM when coasting.

**Comment by cgutman** [comment, marked as answer]
>  The specific service using the RAM seems to be related to the WSL

What do you mean by this statement? Sunshine has nothing to do with the Windows Subsystem for Linux. We don't depend on it or interact with it in any way.

What do you actually see in RAM usage of the Sunshine.exe process? Sunshine.exe and SunshineSvc.exe are the only processes that we create on the system.

On my system, Sunshine.exe is idling at 6 MB of RAM. It will vary slightly depending on your GPU drivers, OS version, etc, but it should be in the low tens of megabytes at maximum.

**Reply by Travillion** [reply]
My apologies. It is actually related to Docker Desktop. The way it was positioned in Task Manager led me to believe it was a sub-process of Sunshine.

**Comment by Travillion** [comment]
Thank you for your response. I suspected WSL was involved because the
actual process name ended in WSL. In task manager, it appeared to be a sub
process of Sunshine. I am away from my computer right now but I will
provide more details when I can get back to my computer.
